### ISSUE TITLE
[Question] Would you consider publishing this to maven central?

### ISSUE THREAD

**Issue by lewisd32** [Yleisradio/aws-maven#8]
Really thrilled with how this is working for us, but would love to be able to use it without having to add a repository to our poms who's reliability is unknown to us. Would you consider deploying it to maven central?

**Comment by huima** [comment]
Sure. We just have not put any effort into it, as it has not been a need for us and we really have not been interested to go through the hoops of getting it published yet.

I'll add a note to our board about it and talk about it with @milankinen

**Comment by milankinen** [comment]
Yeah I don't see any problem at all. I added task to our internal board, shouldn't be a big one.

**Comment by walery** [comment]
Can you give some updates? :)

**Comment by milankinen** [comment]
Ping @huima ?

**Comment by lewisd32** [comment]
Still really interested in this. Any update?

**Comment by lewisd32** [comment]
Is there anything I can do to help make this happen?

**Comment by walery** [comment]
For all that are still interested - I've [forked the repository](https://github.com/2iq/aws-maven) and [publish it on maven central](https://mvnrepository.com/artifact/com.x2iq.tools/aws-maven/1.4.4).
Version 1.4.4 on maven central is same as 1.4.2 on yle repo.
